### ISSUE TITLE
pkg-config proj does not declare location of installed proj data

### ISSUE THREAD

**Issue by rsbivand** [OSGeo/PROJ#1307]
It would be helpful if proj.pc declared where the contents of data are put (e.g. /usr/local/share/proj). It would also be helpful to remind people to put proj-datumgrid contents there, or to go back to previous behaviour and actually install from data/ in the source tree. `pj_open_lib()` used to work < 6 without PROJ_LIB set, and to set this, proj.pc declaring the installed location would be very useful in writing configure files.

**Comment by kbevers** [comment]
I think this is the first time I've heard of anyone actually using the `proj.pc` that we supply. Can you submit a PR with you suggestions?

Regarding the installation of grids it is documentented here: https://proj4.org/install.html#autotools and in the README. If they are not up to date they should be adjusted accordingly.

**Comment by rsbivand** [comment]
Sorry, I was referring to configure files in R **sf** and **rgdal**, etc., not the proj configure file. See for example https://github.com/r-spatial/sf/blob/master/configure.ac around line 293:

PROJ_CONFIG="pkg-config proj"

if `$PROJ_CONFIG --exists` ; then
  AC_MSG_NOTICE([pkg-config proj exists, will use it])
  proj_config_ok=yes
else
  proj_config_ok=no
fi

And yes, we do use it!

**Comment by rsbivand** [comment]
And after better checking, `pj_open_lib()` does work in 6.0.0 without PROJ_LIB set. We've previously checked that conus is present to see whether proj-datumgrid is unpacked in the data directory.

**Comment by kbevers** [comment]
Well now I am just confused... is this about changes to be made in sf rgdal?

**Comment by rsbivand** [comment]
No, it is about proj.pc declaring the data directory (as GDAL does with `gdal-config --datadir`), but maybe it isn't important. What I thought might be helpful was to show the target data directory into which a zipfile should be unpacked in a message from configure in sf or rgdal. It'll be a little while before packagers provide packages for the data directories, so users may also need to do this themselves, and if we can give them the directory path, they'll do it themselves instead of asking.

**Comment by kbevers** [comment]
Ah, I understand now. If it is possible to make `proc.pc` do that I don't see a problem with it. I don't know how to do that myself though. An alternative solution would be for you to just derive the placement of the data files from your own installation settings. It should be straight forward to find the data dir from the installation prefix.

**Comment by rsbivand** [comment]
Yes, but not on other people's installations, where we have no control of where they put stuff. @rouault do you recall how to put datadir in a file that pkg-config exposes? GDAL does, but it doesn't seem to show it, of course gdal-config does.

**Comment by cjmayo** [comment]
Hope I have understood this. For gdal:
```
$ pkg-config --variable=datadir gdal
/usr/share/gdal
```

 #1348 adds this for PROJ:
```
$ pkg-config --variable=datadir proj
/usr/share/proj
```

**Comment by kbevers** [comment]
Fixed with #1348
